src/Pos/Generator/Block/Mode.hs

```diff
@@ -123,7 +124,7 @@ data BlockGenContext = BlockGenContext
     , bgcSystemStart       :: !Timestamp
     , bgcParams            :: !BlockGenParams
     , bgcDelegation        :: !DelegationVar
-    , bgcGenesisUtxo       :: !GenesisUtxo
+    , bgcGenesisTxpContext :: !GenesisTxpContext
```

Comment:
I think it's wrong, it should have only `GenesisStakeholders`. I don't see why `GenesisUtxo` is needed. Moreover, it seems dangerous to have it in the context, because someone can accidentally use it, but it's wrong.
Maybe I am missing smth tho.